packages/jsapi-utils/src/SessionUtils.ts

```diff
@@ -24,18 +24,6 @@ export interface SessionWrapper {
   dh: typeof DhType;
 }
 
-/**
- * @returns New connection to the server
- */
-export function createConnection(
-  dh: typeof DhType,
-  websocketUrl: string
-): DhType.IdeConnection {
-  log.info(`Starting connection to '${websocketUrl}'...`);
-
-  return new dh.IdeConnection(websocketUrl);
```

Comment:
`IdeConnection` is a protected constructor which now shows a ts error in the 0.37+ types. The wrapping connection doesn't appear to be called anywhere, so deleting it. 

Per @niloc132 this call shouldn't have worked as of early 2023.